app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/operations.rb

```diff
@@ -9,12 +9,6 @@ def raw_set_custom_field(attribute, value)
     run_command_via_parent(:vm_set_custom_field, :attribute => attribute, :value => value)
   end
 
-  included do
-    supports :terminate do
-      unsupported_reason_add(:terminate, unsupported_reason(:control)) unless supports?(:control)
-    end
-  end
-
```

Comment:
This is implemented slightly differently now in Vm and Template subclasses.  Specifically Templates don't need a power-state check and they can't use `supports?(:control)` directly